models/br_bd_metadados/available_options.sql

```diff
@@ -0,0 +1,5 @@
+SELECT
```

Comment:
Melhor deixar tudo em inglês, não?

Reply:
Se for mudar para inglês não esquecer de mudar também no `schema.yml`